drivers/SmartThings/jbl/src/jbl/device_manager.lua

```diff
@@ -0,0 +1,145 @@
+local log = require "log"
+local json = require "st.json"
+local utils = require "st.utils"
+local fields = require "fields"
+local discovery = require "discovery"
+
+local capabilities = require "st.capabilities"
+
+local device_manager = {}
+device_manager.__index = device_manager
+
+local function is_new_audioTrackData(device, audioTrackData)
+    local latestTrackData = device:get_latest_state("main", capabilities.audioTrackData.ID, capabilities.audioTrackData.audioTrackData.NAME, "")
+
+    if not latestTrackData or latestTrackData == "" then
+        return true
+    end
+
+    if (audioTrackData.title == latestTrackData.title)
+        and (audioTrackData.artist == latestTrackData.artist)
+        and (audioTrackData.album == latestTrackData.album)
+        and (audioTrackData.albumArtUrl == latestTrackData.albumArtUrl)
+        and (audioTrackData.mediaSource == latestTrackData.mediaSource) then
+        return false
+    end
+
+    return true
+end
+
+local jbl_playback_state_to_smartthings_playbck_status_table = {
```

Comment:
Just for the sake of clarity, the profile YAML for the speaker says that it only supports "playing" and "paused" so do we need the other elements in the table here? If the driver tries to emit an event for something other than "playing" or "paused" it will get rejected before it makes it to the cloud platform because of the profile.